hpx/runtime/threads/thread_enums.hpp

```diff
@@ -68,15 +66,21 @@ namespace hpx { namespace threads
     /// thread is being restarted
     enum thread_state_ex_enum
     {
-        wait_unknown = -1,
-        wait_signaled = 0,  ///< The thread has been signaled
-        wait_timeout = 1,   ///< The thread has been reactivated after a timeout
-        wait_terminate = 2, ///< The thread needs to be terminated
-        wait_abort = 3      ///< The thread needs to be aborted
+        wait_unknown = 0,
+        wait_signaled = 1,  ///< The thread has been signaled
+        wait_timeout = 2,   ///< The thread has been reactivated after a timeout
+        wait_terminate = 4, ///< The thread needs to be terminated
+        wait_abort = 4      ///< The thread needs to be aborted
```

Comment:
This doesn't look right, why is wait_terminate and wait_abort the same numerical value?